CHANGES.rst

```diff
@@ -1,6 +1,13 @@
 Changes
 =======
 
+N.N.N (YYYY-MM-DD)
+------------------
+
+* ``scrapy-zyte-api[provider]`` now requires zyte-common-items >= 0.20.0.
```

Comment:
I wonder how big is it an issue, and if it worths it to have the feature disabled for older zyte-common-items versions, instead of raising an exception. @VMRuiz any opinion on that?

Reply:
> What would be the reason for this not to be compatible with older `zyte-common-items` packages?

Mainly the new `is_auto_field` method we added in 0.20.0.

Although before that I also had increased the version, to 0.19.0, I think because of the imports that the feature needed, of all the `Auto` classes.

We could make it so that the stat only works on zyte-common-items >= 0.20.0, but earlier versions can be installed, would that work?

If not, we can try to support earlier versions as well for the stat, it simply complicates the implementation a bit.

Reply:
I was worried that some data types changed, but so far, based on the changelog, it seems zyte-common-items don't have any breaking changes, or at least nothing super-visible. But let's wait for @VMRuiz 's feedback, his opinion matters most here.

Reply:
From my side, I haven't noticed any conflicts. I checked requirements in the first projects we would be using this and they match.
(No functional testing performed though).

Reply:
@kmike Should we go ahead with 0.20.0 as minimum, or lower it just to be safe?

Reply:
Let's proceed with 0.20.0; we can add support for older versions later if that turn out to be an issue in practice.